tutorials/colors/colormap-manipulation.py

```diff
@@ -132,12 +132,13 @@ def plot_examples(colormaps):
 plot_examples([viridis, newcmp])
 
 ##############################################################################
-# We can easily reduce the dynamic range of a colormap; here we choose the
-# middle 0.5 of the colormap.  However, we need to interpolate from a larger
-# colormap, otherwise the new colormap will have repeated values.
+# We can reduce the dynamic range of a colormap; here we choose the
+# middle 0.5 of the colormap.  Note, however, that because viridis is a
+# listed colormap, some colors will be repeated when doing the
+# nearest-neighbour interpolation.
```

Comment:
```suggestion
# We can reduce the dynamic range of a colormap; here we choose the
# middle half of the colormap.  Note, however, that because viridis is a
# listed colormap, we will end up with 128 discrete values instead of the 256 values
# that were in the original colormap. This method does not interpolate in color-space
# to add new colors.
```